src/napari_cryoet_data_portal/_uri_widget.py

```diff
@@ -31,13 +34,27 @@ def __init__(self, parent: Optional[QWidget] = None) -> None:
         self.setTitle("Portal")
         self._connect_button = QPushButton("Connect")
         self._disconnect_button = QPushButton("Disconnect")
+
         self._uri_edit = QLineEdit(GRAPHQL_URI)
         # Only allow the default portal URI because invalid ones will cause
         # indefinite hangs:
         # https://github.com/chanzuckerberg/cryoet-data-portal/issues/16
         self._uri_edit.setReadOnly(True)
         self._uri_edit.setCursorPosition(0)
         self._uri_edit.setPlaceholderText("Enter a URI to CryoET portal data")
+        
+        filter_ids_layout = QHBoxLayout()
+        filter_ids_layout.setContentsMargins(0, 0, 0, 0)
+        self._filter_ids_type = QComboBox()
+        self._filter_ids_type.addItem("Dataset IDs", Dataset)
+        self._filter_ids_type.addItem("Run IDs", Run)
+        self._filter_ids_type.addItem("Spacing IDs", TomogramVoxelSpacing)
```

Comment:
Can we at least call this VxlSpacing IDs ?  The texting generation will read that ok.  Spacing is not saying much.

Reply:
I originally had this as "Voxel Spacing IDs" and then compressed it to "Spacing IDs" for horizontal space saving, so I'm going to switch back to that.